scripts/run-dev-win.sh

```
@@ -0,0 +1,78 @@
+  cygpath -u "$1"
+}
+
+find_pnpm() {
+  if command -v pnpm >/dev/null 2>&1; then
+    command -v pnpm
+    return 0
+  fi
+  local local_appdata_unix
+  local_appdata_unix="$(to_unix_path "${LOCALAPPDATA:-}")" || return 1
+  local candidate
+  candidate="$(ls -d "$local_appdata_unix"/Microsoft/WinGet/Packages/pnpm.pnpm_* 2>/dev/null | head -n1 || true)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)